src/cf-layout.less

```diff
@@ -583,41 +646,36 @@
     .content-l_col.@{col} {
         display: block;
         width: 100%;
-        
+
         &.content-l_col__before-divider {
             .grid_column__top-divider();
         }
     }
-    
+
     &.content-l__large-gutters {
         .content-l_col.@{col}.content-l_col__before-divider {
             border-left-width: @grid_gutter-width;
         }
     }
-    
-    .content-l_col.@{col} + .content-l_col.@{col} {
+
+    .content-l_col + .@{col} {
```

Comment:
Thanks, I was fighting with the combos and realized we don't really need to manually set them since they all come after another column.